src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Protocol/AutoInsert/RemoteInsertTextEdit.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System.Runtime.Serialization;
+using Microsoft.CodeAnalysis.Text;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+using static Microsoft.VisualStudio.LanguageServer.Protocol.VsLspExtensions;
+using static Roslyn.LanguageServer.Protocol.RoslynLspExtensions;
+using RoslynInsertTextFormat = Roslyn.LanguageServer.Protocol.InsertTextFormat;
+
+namespace Microsoft.CodeAnalysis.Razor.Protocol.AutoInsert;
+
+[DataContract]
+internal readonly record struct RemoteInsertTextEdit(
+        [property: DataMember(Order = 0)]
+        LinePositionSpan LinePositionSpan,
+        [property: DataMember(Order = 1)]
+        string NewText,
+        [property: DataMember(Order = 2)]
+        RoslynInsertTextFormat InsertTextFormat
+    )
+{
```

Comment:
Please move closing parenthesis to the previous line. If it were me, I'd format the parameter list like so:

```suggestion
internal readonly record struct RemoteInsertTextEdit(
    [property: DataMember(Order = 0)] LinePositionSpan LinePositionSpan,
    [property: DataMember(Order = 1)] string NewText,
    [property: DataMember(Order = 2)] RoslynInsertTextFormat InsertTextFormat)
{
```